test/e2e/specs/editor/blocks/paragraph.spec.js

```diff
@@ -1,18 +1,28 @@
 /**
  * WordPress dependencies
  */
-const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+const {
+	test,
+	expect,
+	Editor,
+} = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	editor: async ( { page }, use ) => {
+		await use( new Editor( { page } ) );
+	},
+} );
```

Comment:
I don't think we actually need to do this in every test. Site editor tests can probably do something like:
`editor.useIframeCanvas()` in a `beforeAll`. 🤔

Reply:
Actually, this doesn't seem possible, there's an error if I try this -  `Error: "context" and "page" fixtures are not supported in beforeAll. Use browser.newContext() instead.`

Reply:
Yeah we can't do any of that stuff in `beforeAll` or `afterAll`, that's one of the reasons we need `requestUtils`.